migtests/tests/pg/multiple-schemas/validate

```diff
@@ -49,23 +60,26 @@ def check_validations_per_schema(tgt,schema_name):
 	print(f"fetched extension for extname - {fetched_extension}")
 	assert fetched_extension == 'pgcrypto'
 
-	chk_executed_query = tgt.run_query_and_chk_error("call tt_insert_data(5);", None)
-	print(f"error in executing query - {chk_executed_query}")
-	assert chk_executed_query == False
+	# don't want to insert anything in case of live migration
```

Comment:
why so ? what's the issue?

Reply:
1. We don't apply sequences until cutover. 
2. It gets confusing imo. Better to constrain the IUDs to snapshot_data or *_delta files.

Reply:
Yeah, fine. It is fine anyway we have these validations in `validateAfterChanges`.